modules/rosa-architecture.adoc

```diff
@@ -10,4 +10,12 @@ Red Hat manages the cluster with limited access through a public network. For mo
 .ROSA deployed on public and private networks
 image::156_OpenShift_ROSA_Arch_0621_arch.svg[ROSA deployed on public and private networks]
 
-Alternatively, install a cluster using AWS PrivateLink, which is hosted on private subnets only.
+If you are using {hcp-title-first}, you can create your clusters on public and private networks as well. Those architectures look like the following images.
```

Comment:
Instead of "Those architectures look like the following images", maybe something like  "The architecture of both public and private networks are displayed in the images below"?